.github/workflows/release.yaml

```diff
@@ -1,47 +0,0 @@
-
-
-name: Release Charts
```

Comment:
why did you remove this? I could use the deployed version of master to unblock myself with https://github.com/acryldata/datahub-helm/issues/221

Reply:
This was removed temporarily until this fix is validated in a staging environment. I believe this would be restored when tested. I assume there wasn't an easy way to temporarily disable this.